styles.css

```
@@ -10866,110 +10872,29 @@ body.modal-open {
+  border-radius: 9999px !important;
+  font-size: 14px !important;
+  font-weight: 600 !important;
+  line-height: 1 !important;
 
-.search-section {
-  display: flex;
-  gap: 10px;
-  flex: 1;
-  min-width: 250px;
-}
+  display: inline-flex !important;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)